packages/metro/src/ModuleGraph/worker/collectDependencies.js

```diff
@@ -199,6 +209,16 @@ function collectDependencies<TSplitCondition = void>(
         return;
       }
 
+      if (
+        callee.type === 'MemberExpression' &&
+        (callee.object || {}).name === 'require' &&
+        (callee.property || {}).name === 'context'
```

Comment:
Please also check:
1. That `object` and `property` are identifiers
2. That `computed` is false
3. That `require` refers to the global and not something else. (`!path.scope.getBinding('require')`)